internal/desktop_browser.py

```diff
@@ -514,19 +523,22 @@ def on_stop_capture(self, task):
             else:
                 subprocess.call(['sudo', 'killall', 'tcpdump'])
                 kill_all('tcpdump', False)
-        if self.ffmpeg is not None:
+        if self.ffmpeg is not None and self.video_capture_running:
             logging.debug('Stopping video capture')
             self.video_capture_running = False
             if platform.system() == 'Windows':
                 logging.debug('Attempting graceful ffmpeg shutdown\n')
                 self.ffmpeg.communicate(input='q')
                 if self.ffmpeg.returncode is not 0:
                     logging.exception('ERROR: ffmpeg returned non-zero exit code %s\n', str(self.ffmpeg.returncode))
+                    raise RuntimeError('ffmpeg encountered an error during exit. Video may be corrupt.')
```

Comment:
I'd rather not re-raise the exception without knowing how far up it will go before getting caught.  It may leave other things like traffic-shaping in a bad state.